groupstage.js

```diff
@@ -53,7 +59,7 @@ var makeMatches = function (numPlayers, groupSize, hasAway) {
 // ------------------------------------------------------------------
 
 var GroupStage = Tournament.sub('GroupStage', function (opts, initParent) {
-  var ms = makeMatches(this.numPlayers, opts.groupSize, opts.meetTwice);
+  var ms = makeMatches(this.numPlayers, opts.groupSize, opts.meetTwice, opts.numberOfDuels);
```

Comment:
meetTwice and numberOfDuels are conflicting name options, it's a bit awkward because people can think it's not referring to the home/away system football often uses.

Reply:
Yes. I tried to mantain retro-compatibility adding that extra parameter but the ideal way is to have only numberOfDuels with default to 1